src/components/Atoms/Controls/Input/Input.stories.tsx

```
@@ -59,7 +59,7 @@ export const Password: Story = {
     label: "Password Input",
     placeholder: "Enter password",
   },
-  render: (args) => <InputWrapper {...args} />,
+  render: (args: any) => <InputWrapper {...args} />,
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)